src/components/Start.tsx

```diff
@@ -1,56 +1,61 @@
 import React from 'react';
-import { connect, ConnectedProps } from 'react-redux';
-
-import { RootState, isLoading, startLoading } from '../store';
-
-/**
- * mapState - is a function receiving full Redux state as the first argument
- * and returning an object with extra props that will be added to a component
- * after calling connect(mapState)(MyComponent)
- *
- * @param {object} state - full Redux state
- *
- * @return {object}
- */
-const mapState = (state: RootState) => {
-  return {
-    loading: isLoading(state), // we use a selector `isLoading` defined in the store
-  };
-};
+import { connect } from 'react-redux';
 
-/**
- * We use an object syntax for `mapDispatch` where
- * `load` - is a callback name passed to the component as a prop
- * `startLoading` - is an action creator defined in the store
- */
-const mapDispatch = {
-  load: startLoading,
+import {
+  RootState, startLoading, setTodos, getTodos,
+} from '../store';
+import { getData, TODOS_URL, USERS_URL } from '../Api';
+import { User, Todo, TodoWithUser } from '../interfaces';
+
+type Props = {
+  isLoading: () => (void);
+  initTodos: (todos: TodoWithUser[]) => (void);
 };
 
-/**
- * We split the connect(mapState, mapDispatch)(MyComponent) into 2 parts
- * to be able to use `typeof connector` for `MyComponent` props
- */
-const connector = connect(mapState, mapDispatch);
+const Start: React.FC<Props> = ({ isLoading, initTodos }) => {
+  const getTodoList = async () => {
+    isLoading();
```

Comment:
isLoading это функция? некорректное название для нее
функция должна быть глаголом - сделать что-то

isLoading подойдет для булевской переменной

если эта штука просто меняет лоадинг на !лоадинг, то
```suggestion
    toggleLoading();
```